working-on-data-science-projects.adoc

```diff
@@ -18,7 +18,7 @@ include::_artifacts/document-attributes-global.adoc[]
 [role='_abstract']
 As a data scientist, you can organize your data science work into a single project. A data science project in {productname-short} can consist of the following components:
 
-Workbenches:: Creating a workbench allows you to add a Jupyter notebook to your project.
+Workbenches:: Creating a workbench allows you to work with models in your preferred IDE, such as a Jupyter notebook. 
```

Comment:
```suggestion
Workbenches:: Creating a workbench allows you to work with models in your preferred IDE, such as JupyterLab.
```

https://jupyterlab.readthedocs.io/en/latest/ is what we offer. We don't provide https://jupyter-notebook.readthedocs.io/en/latest/ images.